specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -365,15 +359,15 @@
             },
             "put": {
                 "x-ms-examples": {
-                    "Creates or updates an Office365 data connector.": {
-                        "$ref": "./examples/dataConnectors/CreateOfficeDataConnetor.json"
+                    "Creates or updates an action of alert rule.": {
+                        "$ref": "./examples/actions/CreateActionOfAlertRule.json"
                     }
                 },
                 "tags": [
-                    "Data Connectors"
+                    "Actions"
                 ],
-                "description": "Creates or updates the data connector.",
-                "operationId": "DataConnectors_Create",
+                "description": "Creates or updates the action of alert rule.",
+                "operationId": "AlertRules_CreateAction",
```

Comment:
Should be AlertRules_CreateOrUpdateAction.  Same rule applies to all operationIds with a _Create suffix.